seed/main/hi_Latn.xml

```diff
@@ -512,7 +512,6 @@ annotations.
 			<language type="uz">Uzbek</language>
 			<language type="vai">Vai</language>
 			<language type="ve">Venda</language>
-			<language type="vec">Vietnamese</language>
```

Comment:
oops

Reply:
I would expect that these would cause collisions when inheritance is  included. Is that not happening?

Reply:
i deleted `vec` so that it's no longer an issue

Reply:
Okay, but it would cause collisions for any other instances of the wrong line in the xml, right?

Reply:
@AEApple  this line was deleted